fosite-example/main.go

```diff
@@ -131,6 +131,11 @@ func tokenEndpoint(rw http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	// Grant requested scopes
```

Comment:
Why did you add this? Is it to get the resource owner credentials grant working with scopes?